src/runtime/manifest.ts

```diff
@@ -717,6 +724,15 @@ ${e.message}
     manifest._resources[schemaItem.name] = schemaItem.data;
   }
 
+  private static _processAnnotation(manifest: Manifest, schemaItem: AstNode.AnnotationNode) {
+    const params: Dictionary<SchemaPrimitiveTypeValue> = {};
+    for (const param of schemaItem.params) {
+      params[param.name] = param.type;
+    }
+    manifest._annotations[schemaItem.name] = new Annotation(
```

Comment:
Will we ever want multiple annotations on a single schemaItem?

Reply:
sorry, the naming is confusing (i blame copy&paste), i'm fixing it in the next pr.
this is annotationItem (not schemaItem).
annotation name has to be unique, because it is referenced by its name.